orte/mca/errmgr/base/help-errmgr-base.txt

```diff
@@ -61,9 +62,10 @@ route found between them. Please check network connectivity
 (including firewalls and network routing requirements).
 #
 [node-died]
-ORTE has lost communication with its daemon located on node:
+ORTE has lost communication with a remote daemon.
```

Comment:
That information is not accessible at this level. To see more information you can pass: `-mca oob_base_verbose 7` which will show messages from the `oob/tcp` component (for example) during the connection lost path.
 1. [mca_oob_tcp_peer_close](https://github.com/open-mpi/ompi/blob/master/orte/mca/oob/tcp/oob_tcp_connection.c#L1045) - multiple way to get here, but once we call it then we lose the fine grained context. The verbose output should help pinpoint where this started from.
 1. [mca_oob_tcp_component_lost_connection](https://github.com/open-mpi/ompi/blob/master/orte/mca/oob/tcp/oob_tcp_component.c#L1119)
 1. [proc_errors](https://github.com/open-mpi/ompi/blob/master/orte/mca/errmgr/default_hnp/errmgr_default_hnp.c#L288) then reports the issue and aborts the job.